README.md

```diff
@@ -1,5 +1,7 @@
 # fastify-oauth2
 
+THIS IS [CLONE](https://github.com/gjovanov/fastify-oauth2) FROM: ['fastify-oauth2'](https://github.com/fastify/fastify-oauth2) with support for multiple strategies based on [PR #34](https://github.com/fastify/fastify-oauth2/pull/34) which still is neither accepted nor rejected thus not merged.
```

Comment:
This is so rude.
Are you telling us that we must check your PR in how many hours?

Reply:
Hey @Eomm,
pls don't perceive this rude, because the intention wasn't to be that at all.

I do appreciate your effort and hard work here.

The reason this comment was added is because we needed to move forward with supporting multiple OAuth providers in [this project](https://github.com/gjovanov/roomler), however, we were not sure if this REPO was activly mainened since seeing the previous such PR #17 got stale and in #9 I was asking 21 days ago about any progress on #17.

Also even though in our local repo we have the changes of this PR, with the current verion of the NPM package for this repo, running the tests on our build server are failing after `npm i`. Hence we decided as temp solution to create a clone [NPM package](https://www.npmjs.com/package/fastify-oauth2-multi) that includes this PR.

Again, I do understand that PRs cannot be checked within hours, but pls also understand we don't want to postpone the dev on our project Roomler.

So all this is a temp solution for us that allows us to move forward, until this support for multiple strategies is supported with this repo.

Sorry again if I have caused hard feelings with this. It definitevly wasn't my intention to do so and I hope you also understand our position.

Thanks.

Reply:
Thanks for the clarification

In these cases -for temporary unreleased code-, I would suggest installing directly a github commit: `npm install <githubname>/<githubrepo>[#<commit-ish>]` also

Reply:
WOW, I wasn't aware of this option.

I've reverted those changed from the `README.md` as well as `package.json` and npm installed via github.

Thanks for shedding light on this option